lib/slimmer/processors/tag_mover.rb

```diff
@@ -3,8 +3,8 @@ class TagMover
     def filter(src,dest)
       move_tags(src, dest, 'script', :dest_node => 'body', :keys => ['src', 'inner_html'])
       move_tags(src, dest, 'link',   :must_have => ['href'])
-      move_tags(src, dest, 'meta',   :must_have => ['name', 'content'], :keys => ['name', 'content', 'http-equiv'])
-      move_tags(src, dest, 'meta',   :must_have => ['property', 'content'], :keys => ['property', 'content'])
+      move_tags(src, dest, 'meta',   :must_have => ['name', 'content'], :keys => ['name', 'content', 'http-equiv'], insertion_location: :top)
```

Comment:
We've got :must_have and then insertion_location: - could these be consistent with the colon position?